src/pages/Dashboard/Dashboard.component.tsx

```diff
@@ -10,6 +10,7 @@ import { PageAligner } from '../../components/PageView/PageView.component';
 import { TransactionsTableContainer } from './TransactionsTable/TransactionsTable.container';
 import { BalanceBlockProps, DashboardComponentProps } from './Dashboard.types';
 
+
```

Comment:
Looks good, but one last thing: Are you using prettier? We have `.prettierrc` file with rules that I think your editor is not applying. Maybe you have some overrides? Please check that.